dags/ods/kktix_ticket_orders/udfs/kktix_loader.py

```diff
@@ -43,6 +46,30 @@ def _load_to_bigquery(payload: List[Dict]) -> None:
     job.result()
 
 
+def _load_to_bigquery_dwd(payload: List[Dict]) -> None:
```

Comment:
sorry for the nit picking, but seems to me that this function is more like `transform` than `load`